### DISCUSSION TITLE
Could Tabby can open a terminal in the current directory？

### DISCUSSION THREAD

**Discussion by shanguwei** [Eugeny/tabby#9357] [Q&A]
Could Tabby can open a terminal in the current directory？
os : Ubuntu22.04
I want to use tabby instead of native terminal，but Tabby doesn’t seem to be able to flexibly specify the working directory.
The working directory in the settings is still fixed.
And the new terminal opened by ctrl + shift +n in linux cannot inherit the directory

**Comment by dkebler** [comment]
AFAIK after messing around it won't work from nemo with the default open terminal action unless maybe you write a custom open terminal action.   From the commandline you can do `tabby .`  and that works to open a tab in the current directory so it's just not getting the current directory from the action.

Also it seems you can't run two instances of tabby  (only multiple tabs within).  That's a no go for me.  

So removed it and I just stuck with gnome-terminal
